bundles/specmate-jira-connector/src/com/specmate/connectors/jira/internal/services/JiraConnector.java

```diff
@@ -445,4 +445,10 @@ public String getProjectName() {
 		return projectName;
 	}
 
+	@Override
+	public Requirement getRequirementById(String id) throws SpecmateException {

```

Comment:
please confirm: When the requirement is imported like this the next call to getContainerForRequirement will yield the efault folder?

Reply:
You are right. Is there a way to get the epic for a story so I can add it in the right position?

Reply:
the problem is that the parent of a story need not always be the epic, in specmate you can configure specific jql queries to define parents and children. But you can only define a query "give me the children of the paren" and not the other way round. Either we need a new configuralbe query (which I would not do, as this could cause stragne behavior if they dont match) or we just leave it as it is and rely on the sync mechanism to put in on the right spot. 

An additional point: an imported isse might be out of scope of a configured query ---> will the sync mechanism delete the important item or leave it where it is? This is definitely somoething we should check.

Reply:
At the moment, the out of scope element (e.g Bug) stays in the database and is not deleted after a full sync